### PR TITLE
ports/esp32: Added new UM TinyWATCH S3.

### PR DESCRIPTION
**Pull request by UnexpectedMaker**
Added my new TinyWATCH S3 board

### CODE REVIEW

File: ports/esp32/boards/UM_TINYWATCHS3/sdkconfig.board [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+CONFIG_ESPTOOLPY_FLASHSIZE_4MB=
+CONFIG_ESPTOOLPY_FLASHSIZE_8MB=y
+CONFIG_ESPTOOLPY_FLASHSIZE_16MB=
+CONFIG_SPIRAM_MEMTEST=
+CONFIG_PARTITION_TABLE_CUSTOM=y
+CONFIG_PARTITION_TABLE_CUSTOM_FILENAME="partitions-8MiB.csv"
+
+CONFIG_LWIP_LOCAL_HOSTNAME="UMTinyWATCHS3"
+
+CONFIG_TINYUSB_DESC_CUSTOM_VID=0x303A
```
Comment (dpgeorge) [inline ports/esp32/boards/UM_TINYWATCHS3/sdkconfig.board:14]: To use a custom VID/PID you need to add two extra options as per this commit: 1cd61149e4328154110d4b1e38bcbecdcc0ec386

Comment (UnexpectedMaker) [reply ports/esp32/boards/UM_TINYWATCHS3/sdkconfig.board:14]: Interesting, I had my S3 boards setup the same as I PR'd here and it worked as expected, but I added the extra lines for all of the boards that use a custom PID.

Thanks for the pointer.

**Comment by dpgeorge** [comment]
Thanks for updating, now merged.

**Comment by UnexpectedMaker** [comment]
> Thanks for updating, now merged.

Thanks Damien!

**Pull request merged**